.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  build_docs:
+    docker:
+      - image: circleci/python:3.6-stretch
+    steps:
+      # Get our data and merge with upstream
+      - run: sudo apt-get update
+      - checkout
+      - run: echo $(git log -1 --pretty=%B) | tee gitlog.txt
```

Comment:
does circleci really need all the steps between L10 and L21 just to build a python package? Feels like boilerplate hell :-/

Reply:
no idea haha I'm still debugging that

Reply:
(this is copypasta from another repo...I'm trying to figure out how to get the artifacts to load properly)